.travis.yml

```diff
@@ -6,6 +6,7 @@ go:
 install:
   - make dependencies
   - test -z "$(go fmt ./...)"
+  - test -z "$(golint -min_confidence=1.0 ./modules/host)"
```

Comment:
yeah but then you have to set that variable every time you run make if you want to avoid the lint call.